grassroots-shared/src/dtos/PhoneCanvass/PhoneCanvass.dto.ts

```diff
@@ -141,14 +144,27 @@ export class PhoneCanvassAuthTokenResponseDTO extends createDTOBase(
   token!: string;
 }
 
-export class PhoneCanvasTwilioVoiceCallbackDTO extends createDTOBase(
-  "PhoneCanvasTwilioVoiceCallback",
+export class PhoneCanvasTwilioCallStatusCallbackDTO extends createDTOBase(
+  "PhoneCanvasTwilioCallStatusCallback",
 ) {
-  // We require this to be present, but don't want to use default error handling with twilio responses, so
-  // we mark it optional and manually handle the case where it's missing.
   @IsString()
+  CallSid!: string;
+  // eslint-disable-next-line @darraghor/nestjs-typed/all-properties-are-whitelisted, @darraghor/nestjs-typed/all-properties-have-explicit-defined
+  @TwilioCallStatusDecorator()
+  CallStatus!: TwilioCallStatus;
+
   @IsOptional()
-  conference?: string;
+  CallDuration?: number;
+
+  /**
+   * Convert RFC 2822 timestamp to a ms since epoch
```

Comment:
Is there a reason to convert to epoch? I find RFC 2822 much easier to read. Is it possible we would need timezone in the future?